teps/000X-trigger-crd.md

```diff
@@ -0,0 +1,351 @@
+---
+title: trigger-crd
+authors:
+  - "@kbaig"
+creation-date: 2020-07-14
+last-updated: 2020-07-14
+status: proposed
```

Comment:
im not sure this is technically true since afaik the triggers CRD is already merged? 😬

Reply:
Yes, but it isn't being used by Triggers but cli. So don't what to put there except proposed.

Reply:
I tend to agree with @dibyom and @khrm  :wink:

Reply:
I don't totally agree; fortunately nothing in the controller is operating on the new CRD, but it is committed to the codebase - while we haven't explicitly created an API compatibility policy for our Go libs (https://github.com/tektoncd/pipeline/issues/2748) I think it's probably going to end up being in there. So removing or changing this type would be a breaking change imo

A different way of handling this could have been in a branch or a fork?